dir.props

```diff
@@ -69,13 +69,15 @@
     <DebugSymbols Condition="'$(DebugSymbols)' == ''">true</DebugSymbols>
     <Optimize Condition="'$(Optimize)' == ''">false</Optimize>
     <DebugType Condition="'$(DebugType)' == ''">full</DebugType>
-    <DefineConstants>$(DefineConstants);DEBUG;TRACE</DefineConstants>
+    <DefineConstants Condition="'$(MSBuildProjectExtension)' != '.vbproj'">$(DefineConstants);DEBUG;TRACE</DefineConstants>
```

Comment:
I was hoping that we could keep the general defines non-duplicated by just having one set that is semi-colon delimited and we could split that and then comma delimited it in a single location. Any VB projects should choose to be semi-colon or comma delimited as I think either would still work.